Integrated Help page and updated navbar to show help as well to make …

Eashana0104 opened the pull request:
Now the about page doesn't have any information about help. All the information for help is moved to different HTML file and a link to it is also integrated in the nav bar

danloveg commented on app/recordtransfer/templates/recordtransfer/about.html:
```diff
@@ -14,105 +14,34 @@
     {% blocktrans %}
-    Refer to the following section if you come across a part of the form that you need assistance
-    with.
+    Submissions made through the transfer portal are subject to the maximum submission size.
+    You may upload a <b>maximum of {{ MAX_TOTAL_UPLOAD_COUNT }} files</b>, and <b>at most
+        {{ MAX_TOTAL_UPLOAD_SIZE_MB }} MB of files</b>. Any <b>one file may not exceed
+        {{ MAX_SINGLE_UPLOAD_SIZE_MB }} MB</b>. If you would like to submit <i>more</i> than this
+    amount, please split your submission into multiple parts, and use the group form to group similar
+    submissions together so our archivists know your submissions are related.
```
There is extra unnecessary whitespace at the start of these lines:

```suggestion
    Submissions made through the transfer portal are subject to the maximum submission size.
    You may upload a <b>maximum of {{ MAX_TOTAL_UPLOAD_COUNT }} files</b>, and <b>at most
    {{ MAX_TOTAL_UPLOAD_SIZE_MB }} MB of files</b>. Any <b>one file may not exceed
    {{ MAX_SINGLE_UPLOAD_SIZE_MB }} MB</b>. If you would like to submit <i>more</i> than this
    amount, please split your submission into multiple parts, and use the group form to group similar
    submissions together so our archivists know your submissions are related.
```

danloveg commented on app/recordtransfer/templates/recordtransfer/header.html:
```diff
@@ -1,7 +1,7 @@
 {% load i18n %}
 
-<header class="header {% if request.path == "/" %}homepage{% endif %}">
+<header class="header {% if request.path == " /" %}homepage{% endif %}">
```
Is there a reason for the extra whitespace before `/`?

Eashana0104 replied:
I don't think so, although when I opened it in VS Code, it was just giving a syntax error saying / should be wrapped in single quotes instead of double quotes , so I fixed that

Eashana0104 replied:
yes i am just making a commit for that

danloveg reviewed (changes requested):
Your first pull request, great!

Besides the changes I flagged, I have some other feedback:

- Use the email of your GitHub account for commits, you can set it with `git config --global user.email your@email.com`. Do this before making any more commits! I want your commits to be associated with your GitHub account.
- Set your full name in your git config, like this: `git config --global user.name Your Name`
- Take a quick look at [this guide on writing commit messages](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/)

The name of your commit "[Integrated Help page and updated navbar to show help as well to make navigation possible to help page](https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/548/commits/71adfb6ddf102d209851051832ad1996575e2f1d)" is too long. In general, 72 characters is the suggested maximum commit name length. You can go a little over that but in general it's a good rule to follow. Commit messages should also be *imperative*, almost like you're instructing someone to do that thing (i.e., you'd use the word *integrate* instead of *integrated*). You don't need to rename this commit, but here's a more succinct, imperative name for your commit to give you an idea of how commit messages should look:

> "Create help page and add it to navbar"

Eashana0104 commented:
Hi Daniel,

Thank you for the feedback, I ll look into that. A question though, I don’t have my staff email yet. So, for git can I just use my myumanitoba email or should I wait for my staff email?.

Regards,
Ash

From: Daniel Lovegrove ***@***.***>
Date: Wednesday, May 7, 2025 at 1:44 PM
To: NationalCentreTruthReconciliation/Secure-Record-Transfer ***@***.***>
Cc: Eashana Singhal ***@***.***>, Assign ***@***.***>
Subject: Re: [NationalCentreTruthReconciliation/Secure-Record-Transfer] Integrated Help page and updated navbar to show help as well to make … (PR #548)
Caution! This message was sent from outside the University of Manitoba.


@danloveg requested changes on this pull request.

Your first pull request, great!

Besides the changes I flagged, I have some other feedback:

  *   Use the email of your GitHub account for commits, you can set it with git config --global user.email ***@***.*** Do this before making any more commits! I want your commits to be associated with your GitHub account.
  *   Set your full name in your git config, like this: git config --global user.name Your Name
  *   Take a quick look at this guide on writing commit messages<https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/>

The name of your commit "Integrated Help page and updated navbar to show help as well to make navigation possible to help page<https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/548/commits/71adfb6ddf102d209851051832ad1996575e2f1d>" is too long. In general, 72 characters is the suggested maximum commit name length. You can go a little over that but in general it's a good rule to follow. Commit messages should also be imperative, almost like you're instructing someone to do that thing (i.e., you'd use the word integrate instead of integrated). You don't need to rename this commit, but here's a more succinct, imperative name for your commit to give you an idea of how commit messages should look:

"Create help page and add it to navbar"

________________________________

In app/recordtransfer/templates/recordtransfer/about.html<https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/548#discussion_r2078259407>:

> +    Submissions made through the transfer portal are subject to the maximum submission size.

+    You may upload a <b>maximum of {{ MAX_TOTAL_UPLOAD_COUNT }} files</b>, and <b>at most

+        {{ MAX_TOTAL_UPLOAD_SIZE_MB }} MB of files</b>. Any <b>one file may not exceed

+        {{ MAX_SINGLE_UPLOAD_SIZE_MB }} MB</b>. If you would like to submit <i>more</i> than this

+    amount, please split your submission into multiple parts, and use the group form to group similar

+    submissions together so our archivists know your submissions are related.

There is extra unnecessary whitespace at the start of these lines:
⬇️ Suggested change

-    Submissions made through the transfer portal are subject to the maximum submission size.

-    You may upload a <b>maximum of {{ MAX_TOTAL_UPLOAD_COUNT }} files</b>, and <b>at most

-        {{ MAX_TOTAL_UPLOAD_SIZE_MB }} MB of files</b>. Any <b>one file may not exceed

-        {{ MAX_SINGLE_UPLOAD_SIZE_MB }} MB</b>. If you would like to submit <i>more</i> than this

-    amount, please split your submission into multiple parts, and use the group form to group similar

-    submissions together so our archivists know your submissions are related.

+    Submissions made through the transfer portal are subject to the maximum submission size.

+    You may upload a <b>maximum of {{ MAX_TOTAL_UPLOAD_COUNT }} files</b>, and <b>at most

+    {{ MAX_TOTAL_UPLOAD_SIZE_MB }} MB of files</b>. Any <b>one file may not exceed

+    {{ MAX_SINGLE_UPLOAD_SIZE_MB }} MB</b>. If you would like to submit <i>more</i> than this

+    amount, please split your submission into multiple parts, and use the group form to group similar

+    submissions together so our archivists know your submissions are related.

________________________________

In app/recordtransfer/templates/recordtransfer/header.html<https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/548#discussion_r2078277173>:

> @@ -1,7 +1,7 @@

 {% load static %}

 {% load i18n %}



-<header class="header {% if request.path == "/" %}homepage{% endif %}">

+<header class="header {% if request.path == " /" %}homepage{% endif %}">

Is there a reason for the extra whitespace before /?

—
Reply to this email directly, view it on GitHub<https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/548#pullrequestreview-2822827221>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AZM6RSNXJE5YPOAQBTMM4GD25JIB3AVCNFSM6AAAAAB4UGVBVWVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDQMRSHAZDOMRSGE>.
You are receiving this because you were assigned.Message ID: ***@***.***>

danloveg commented:
> Hi Daniel, Thank you for the feedback, I ll look into that. A question though, I don’t have my staff email yet. So, for git can I just use my myumanitoba email or should I wait for my staff email?. Regards, Ash

Use the email linked to your GitHub account, which should be your personal email or whatever email is linked to your account https://github.com/Eashana0104.

Also, I'd suggest replying to messages directly in GitHub, not via email. There's a bunch of junk added to the ends of messages when you respond via email - using the GitHub interface to reply to messages is preferred.

danloveg commented on app/recordtransfer/templates/recordtransfer/base.html:
```diff
@@ -3,55 +3,59 @@
+    <link rel="stylesheet" href="{% static 'base.css' %}">
+    {% endblock stylesheet %}
+    {# Favicon #}
+    <link rel="shortcut icon" type="image/png" href="{% static " recordtransfer/img/favicon.ico" %}" />
```
This is not an bug you introduced, but I noticed the favicon isn't loading because of the quotes here, can you update this line?

```suggestion
    <link rel="shortcut icon" type="image/png" href="{% static 'recordtransfer/img/favicon.ico' %}" />
```

Eashana0104 replied:
I just commited the change

danloveg reviewed (approved):
Great work, everything's working as expected.

---

By the way, this is somewhat related: I don't love how the auto-formatting in VSCode is updating the Django HTML templates, so I opened issue #549 to address that.
